### DISCUSSION TITLE
部分Andorid机型遇到调用了播放但是屏幕没显示动画的问题

### DISCUSSION THREAD

**Discussion by snn-hp** [Tencent/libpag#2367] [Q&A] [answered]
请教一下各位：
现在 有一款荣耀畅玩30plus 机型 遇到调用了播放方法之后，资源加载了，onAnimationStart回调了，onAnimationEnd也回调了，
但是屏幕没显示动画的问题；并且 onAnimationStart 和 onAnimationEnd 的回调仅间隔了几毫秒；也没有报异常；
日志有限只有相关播放流程的日志输出；
有咩有遇到类似现象的？？？？

**Comment by domchen** [comment, marked as answer]
这个情况一般是这个台手机在开发者模式里关闭了动画功能。要在那边重新开启一下，不然连屏幕刷新的回调都没有。目前Android系统的这个调试功能没有地方取消，用户打开了经常就忘记了，只能用户手动取消。否则就是所有动画都无法播放。但是你能做检测并提醒，就是参考PAGView里的代码，animationScale如果等于0的时候，就是设备无法播放动画。
